docusaurus/docs/iOS/components/avatar.md

```diff
@@ -0,0 +1,42 @@
+---
+title: ChatAvatarView
+---
```

Comment:
Isn't the "Properties" section missing from this doc?

Reply:
there is nothing useful there :)